install_config/install/host_preparation.adoc

```diff
@@ -115,6 +115,14 @@ endif::[]
 [[installing-base-packages]]
 == Installing Base Packages
 
+[NOTE]
+====
+Once you have a working inventory file, you can use
+*_/usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml_* to help
+with some prerequisites, such as installing base packages and container storage
+configuration.
```

Comment:
Yeah, as @jianlinliu pointed out these playbooks are not capable of configuring container storage. The will install container runtimes if you don't require any additional configuration. Maybe something like
```
Once you have a working inventory file, you can use
*_/usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml_* to install
container runtimes in their default configuration. If you require customization to
the container runtime please read and follow the directions in this chapter.
```